symphony-bdk-core/src/test/java/com/symphony/bdk/core/config/BdkConfigLoaderTest.java

```diff
@@ -112,4 +117,19 @@ void loadClientGlobalConfig() throws BdkConfigException {
         assertEquals(config.getSessionAuth().getPort(), 8443);
         assertEquals(config.getSessionAuth().getContext(), "context");
     }
+
+    //@Test
+    // CircleCI does not allow to create file in the home directory
```

Comment:
perhaps an assume checking for CircleCI could be used to ignore it conditionally

Reply:
Using this https://junit.org/junit5/docs/current/user-guide/#writing-tests-conditional-execution you mean?

Reply:
Yep :) there are good chances that we can use https://junit.org/junit5/docs/current/api/org.junit.jupiter.api/org/junit/jupiter/api/condition/DisabledIfEnvironmentVariable.html with CIRCLECI
https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables